[5.4] Maintenance Task: Change feature request issue template (#47411)
joomla/joomla-cms: .github/ISSUE_TEMPLATE/Feature_request.md

---
name: Feature request
about: Suggest an idea for this project

---

> Feature request should **NOT** be reported on this repository.

Reviewer: ```suggestion
Feature requests should **NOT** be submitted as an issue.
```

> The correct place for a feature request is the Discussion area. A more complete documentation how we handle feature request can be found here: https://github.com/joomla/joomla-cms/discussions/46934

Reviewer: ```suggestion
The correct place for a feature request is the Discussion area. More in depth information on the feature request process can be found here: https://github.com/joomla/joomla-cms/discussions/46934
```
